src/app/auth/auth.resolver.ts

```diff
@@ -37,22 +38,26 @@ export class AuthResolver {
 
   @Throttle(4, 60)
```

Comment:
what does 4 and 60 mean here?
This is used in multiple places, can we put these into a constant file and resuse the values?

Reply:
yes thats a good idea, will extract them in a separate file. It throttles requests 4 times per 60seconds